src/reprojection.hpp

```diff
@@ -35,28 +45,22 @@ class reprojection
      * Reproject from the source projection lat/lon (EPSG:4326)
      * to target projection.
      */
-    virtual osmium::geom::Coordinates reproject(osmium::Location loc) const = 0;
+    virtual Coordinates reproject(osmium::Location loc) const = 0;
 
     /**
-     * Converts coordinates from target projection to tile projection (EPSG:3857)
+     * Converts coordinates from target projection to tile projection
+     * (EPSG:3857)
      *
-     * Do not confuse with coords_to_tile which does *not* calculate coordinates in the
-     * tile projection, but tile coordinates.
+     * Do not confuse with coords_to_tile which does *not* calculate
+     * coordinates in the tile projection, but tile coordinates.
```

Comment:
Remove this comment now that coords_to_tile has gone away?

Reply:
Adressed in one of the commits.